### PR TITLE
Update About.js

### PR DESCRIPTION
**Pull request by jaimemears**
added {' '}

### CODE REVIEW

**Review by adamjarling — COMMENTED** [review]
@jaimemears I think technically as part of the IIIF manifest spec, the `id` field is supposed to refer to another IIIF resource instead of an arbitrary URL.   So we could go one of 2 ways....

We can stay with convention, and I create a new folder under the `/public/iiif` directory representing the new collections you've added.  Under this new folder will be a single `info.json` file which will contain the URL you'd like the link to point to.

Or, I can update the code which extracts Collection info to see if it's a full URL (ie. https://yaddayadda.com), and if so just link directly.

Thoughts on what may be easiest moving forward?

File: public/iiif/speculative-annotations-manifest.json [hunk 1]
```diff
@@ -53,6 +53,7 @@
       ],
       "partOf": [
         {
+          "id": "https://www.loc.gov/search/?fa=contributor:muybridge,+eadweard&sp=1",
```
Comment (adamjarling) [inline public/iiif/speculative-annotations-manifest.json:56]: @jaimemears Here's the `id` field I was referring to in the other comment

Comment (jaimemears) [reply public/iiif/speculative-annotations-manifest.json:56]: I see. I really dont know. I think whatever you think we should do is best. Second one seems easier?

Comment (adamjarling) [reply public/iiif/speculative-annotations-manifest.json:56]: Gotcha.  Yea, I'll go route 2.

**Comment by jaimemears** [comment]
ok sounds good.

**Comment by adamjarling** [comment]
Merging this locally

**Pull request closed without merging**